src/test/java/ActionTests.java

```diff
@@ -0,0 +1,36 @@
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.AllSongsPage;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class ActionTests extends BaseTest {
+    @Test
+    public void playSong() throws InterruptedException{
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        AllSongsPage songsPage = new AllSongsPage(driver);
+        //Login
+        loginPage.provideEmail("evgeniia.shitikova@testpro.io");
+        loginPage.providePassword("TridY6F2");
+        loginPage.clickSubmit();
+        homePage.chooseAllSongsList();
+        songsPage.contextClickFirstSong();
+        songsPage.choosePlayOption();
+        Thread.sleep(2000);
+        //Assertion
+        Assert.assertTrue(songsPage.isSongPlaying());
+    }
+    @Test
+    public void hoverOverPlayBtn() throws InterruptedException{
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.